rosetta/text/vw_helpers.py

```diff
@@ -89,13 +95,15 @@ def parse_lda_topics(topics_file, num_topics, normalize=True):
         # Once we detect that we're in the valid rows, there better not be
         # any exceptions!
         in_valid_rows = False
-        for line in open_file:
+        for i, line in enumerate(open_file):
             try:
                 # If this row raises an exception, then it isn't a valid row
                 # Sometimes trailing space...that's the reason for split()
                 # rather than csv.reader or a direct pandas read.
                 split_line = line.split()
                 hash_val = int(split_line[0])
+                if hash_val > max_token_hash:
```

Comment:
yeah i had that at first, but is that any different in practice than having default max_token_hash very large? you are basically saying that 

```python
max_token_hash is not None
```

is faster to evaluate than a numeric inequality or that it's just easier to understand. 

Happy to make the change...